handlers/views_handler_filter_string.inc

```diff
@@ -242,7 +242,7 @@ class views_handler_filter_string extends views_handler_filter {
     foreach ($matches as $match) {
       $phrase = false;
       // Strip off phrase quotes
-      if ($match[2]{0} == '"') {
+      if ($match[2][0] == '"') {
```

Comment:
Thanks. Yes. This will throw a fatal error on PHP8.

Reply:
Since the rest of this PR is unnecessary (although, harmless!) I ended up just committing this change in https://github.com/d6lts/views/commit/e2f15e57ee953f92b6bfd04360f6043dfb1cd90c